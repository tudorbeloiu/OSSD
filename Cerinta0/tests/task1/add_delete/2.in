40
1
3
86
187
72
187
34
157
3
72
3
34
3
86
1
3
98
464
133
240
80
60
3
80
3
133
3
98
1
2
176
59
116
392
3
116
1
9
150
376
75
145
172
128
185
450
94
330
179
393
93
57
89
408
178
417
3
185
1
2
122
338
224
66
3
224
3
93
1
8
250
24
191
387
126
140
3
438
163
284
109
119
30
302
1
385
1
3
54
208
143
491
188
281
3
176
3
3
1
4
201
327
45
101
151
64
13
246
3
179
3
150
3
143
1
6
205
370
138
382
23
313
239
254
84
492
149
38
1
4
161
33
196
472
66
367
246
41
3
191
3
89
3
23
3
138
1
5
252
254
143
161
193
188
70
153
50
295
1
2
133
41
140
152
3
50
1
3
104
239
213
88
59
87
1
8
78
207
173
271
11
280
190
292
170
487
156
144
219
220
208
386
3
193
3
1
3
94
3
140
1
4
108
198
90
451
39
283
3
290
3
156
