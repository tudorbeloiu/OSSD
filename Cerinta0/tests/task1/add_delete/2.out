86: (0, 23)
72: (24, 47)
34: (48, 67)
86: (0, 23)
34: (48, 67)
86: (0, 23)
98: (0, 57)
133: (58, 87)
80: (88, 95)
98: (0, 57)
133: (58, 87)
98: (0, 57)
176: (0, 7)
116: (8, 56)
176: (0, 7)
150: (8, 54)
75: (55, 73)
172: (74, 89)
185: (90, 146)
94: (147, 188)
179: (189, 238)
93: (239, 246)
89: (247, 297)
178: (298, 350)
176: (0, 7)
150: (8, 54)
75: (55, 73)
172: (74, 89)
94: (147, 188)
179: (189, 238)
93: (239, 246)
89: (247, 297)
178: (298, 350)
122: (90, 132)
224: (133, 141)
176: (0, 7)
150: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
94: (147, 188)
179: (189, 238)
93: (239, 246)
89: (247, 297)
178: (298, 350)
176: (0, 7)
150: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
94: (147, 188)
179: (189, 238)
89: (247, 297)
178: (298, 350)
250: (133, 135)
191: (351, 399)
126: (400, 417)
3: (418, 472)
163: (473, 508)
109: (509, 523)
30: (524, 561)
1: (562, 610)
54: (611, 636)
143: (637, 698)
188: (699, 734)
150: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
94: (147, 188)
179: (189, 238)
89: (247, 297)
178: (298, 350)
191: (351, 399)
126: (400, 417)
3: (418, 472)
163: (473, 508)
109: (509, 523)
30: (524, 561)
1: (562, 610)
54: (611, 636)
143: (637, 698)
188: (699, 734)
150: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
94: (147, 188)
179: (189, 238)
89: (247, 297)
178: (298, 350)
191: (351, 399)
126: (400, 417)
163: (473, 508)
109: (509, 523)
30: (524, 561)
1: (562, 610)
54: (611, 636)
143: (637, 698)
188: (699, 734)
201: (418, 458)
45: (459, 471)
151: (0, 7)
13: (735, 765)
151: (0, 7)
150: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
94: (147, 188)
89: (247, 297)
178: (298, 350)
191: (351, 399)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
1: (562, 610)
54: (611, 636)
143: (637, 698)
188: (699, 734)
13: (735, 765)
151: (0, 7)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
94: (147, 188)
89: (247, 297)
178: (298, 350)
191: (351, 399)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
1: (562, 610)
54: (611, 636)
143: (637, 698)
188: (699, 734)
13: (735, 765)
151: (0, 7)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
94: (147, 188)
89: (247, 297)
178: (298, 350)
191: (351, 399)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
1: (562, 610)
54: (611, 636)
188: (699, 734)
13: (735, 765)
205: (8, 54)
138: (189, 236)
23: (637, 676)
239: (766, 797)
84: (798, 859)
149: (136, 140)
161: (141, 145)
196: (860, 918)
66: (919, 964)
246: (237, 242)
151: (0, 7)
205: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
149: (136, 140)
161: (141, 145)
94: (147, 188)
138: (189, 236)
246: (237, 242)
89: (247, 297)
178: (298, 350)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
1: (562, 610)
54: (611, 636)
23: (637, 676)
188: (699, 734)
13: (735, 765)
239: (766, 797)
84: (798, 859)
196: (860, 918)
66: (919, 964)
151: (0, 7)
205: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
149: (136, 140)
161: (141, 145)
94: (147, 188)
138: (189, 236)
246: (237, 242)
178: (298, 350)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
1: (562, 610)
54: (611, 636)
23: (637, 676)
188: (699, 734)
13: (735, 765)
239: (766, 797)
84: (798, 859)
196: (860, 918)
66: (919, 964)
151: (0, 7)
205: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
149: (136, 140)
161: (141, 145)
94: (147, 188)
138: (189, 236)
246: (237, 242)
178: (298, 350)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
1: (562, 610)
54: (611, 636)
188: (699, 734)
13: (735, 765)
239: (766, 797)
84: (798, 859)
196: (860, 918)
66: (919, 964)
151: (0, 7)
205: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
149: (136, 140)
161: (141, 145)
94: (147, 188)
246: (237, 242)
178: (298, 350)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
1: (562, 610)
54: (611, 636)
188: (699, 734)
13: (735, 765)
239: (766, 797)
84: (798, 859)
196: (860, 918)
66: (919, 964)
252: (189, 220)
143: (243, 263)
193: (264, 287)
70: (351, 370)
50: (637, 673)
133: (221, 226)
140: (371, 389)
151: (0, 7)
205: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
149: (136, 140)
161: (141, 145)
94: (147, 188)
252: (189, 220)
133: (221, 226)
246: (237, 242)
143: (243, 263)
193: (264, 287)
178: (298, 350)
70: (351, 370)
140: (371, 389)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
1: (562, 610)
54: (611, 636)
188: (699, 734)
13: (735, 765)
239: (766, 797)
84: (798, 859)
196: (860, 918)
66: (919, 964)
104: (637, 666)
213: (667, 677)
59: (678, 688)
78: (965, 990)
173: (0, 0)
11: (0, 0)
190: (0, 0)
170: (0, 0)
156: (991, 1008)
219: (0, 0)
208: (0, 0)
151: (0, 7)
205: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
149: (136, 140)
161: (141, 145)
94: (147, 188)
252: (189, 220)
133: (221, 226)
246: (237, 242)
143: (243, 263)
178: (298, 350)
70: (351, 370)
140: (371, 389)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
1: (562, 610)
54: (611, 636)
104: (637, 666)
213: (667, 677)
59: (678, 688)
188: (699, 734)
13: (735, 765)
239: (766, 797)
84: (798, 859)
196: (860, 918)
66: (919, 964)
78: (965, 990)
156: (991, 1008)
151: (0, 7)
205: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
149: (136, 140)
161: (141, 145)
94: (147, 188)
252: (189, 220)
133: (221, 226)
246: (237, 242)
143: (243, 263)
178: (298, 350)
70: (351, 370)
140: (371, 389)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
54: (611, 636)
104: (637, 666)
213: (667, 677)
59: (678, 688)
188: (699, 734)
13: (735, 765)
239: (766, 797)
84: (798, 859)
196: (860, 918)
66: (919, 964)
78: (965, 990)
156: (991, 1008)
151: (0, 7)
205: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
149: (136, 140)
161: (141, 145)
252: (189, 220)
133: (221, 226)
246: (237, 242)
143: (243, 263)
178: (298, 350)
70: (351, 370)
140: (371, 389)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
54: (611, 636)
104: (637, 666)
213: (667, 677)
59: (678, 688)
188: (699, 734)
13: (735, 765)
239: (766, 797)
84: (798, 859)
196: (860, 918)
66: (919, 964)
78: (965, 990)
156: (991, 1008)
151: (0, 7)
205: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
149: (136, 140)
161: (141, 145)
252: (189, 220)
133: (221, 226)
246: (237, 242)
143: (243, 263)
178: (298, 350)
70: (351, 370)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
54: (611, 636)
104: (637, 666)
213: (667, 677)
59: (678, 688)
188: (699, 734)
13: (735, 765)
239: (766, 797)
84: (798, 859)
196: (860, 918)
66: (919, 964)
78: (965, 990)
156: (991, 1008)
108: (146, 170)
90: (0, 0)
39: (562, 597)
3: (0, 0)
151: (0, 7)
205: (8, 54)
75: (55, 73)
172: (74, 89)
122: (90, 132)
250: (133, 135)
149: (136, 140)
161: (141, 145)
108: (146, 170)
252: (189, 220)
133: (221, 226)
246: (237, 242)
143: (243, 263)
178: (298, 350)
70: (351, 370)
126: (400, 417)
201: (418, 458)
45: (459, 471)
163: (473, 508)
109: (509, 523)
30: (524, 561)
39: (562, 597)
54: (611, 636)
104: (637, 666)
213: (667, 677)
59: (678, 688)
188: (699, 734)
13: (735, 765)
239: (766, 797)
84: (798, 859)
196: (860, 918)
66: (919, 964)
78: (965, 990)
